joplin/api/schema.py

```diff
@@ -425,6 +436,12 @@ def resolve_locations(self, info):
 
         return repr_locations
 
+    def resolve_page_type(self, info):
```

Comment:
do we need this in the API? I know that janis_urls uses it but does janis?

Reply:
this is probably a good question for @chiaberry, I know we have page type resolvers in other places now too, do we need them to build janis?

Reply:
It doesn't look like we need it anymore, but it doesn't hurt either.

Reply:
definitely more of a "can we remove something we need to copypasta around at some point" question

Reply:
not something to change now